src/main/resources/hudson/plugins/promoted_builds/PromotedBuildAction/index.jelly

```diff
@@ -35,14 +35,17 @@
         <h4>Status</h4>
         <div>
           <!-- JENKINS-20492: show re-execute event when manually approved -->
-		      <j:choose>
-		        <j:when test="${it.getPromotionProcess(p.name)!=null and it.canPromote()}">
-		          <form style="float:right" method="post" action="${p.name}/build">
-		            <f:submit value="${%Re-execute promotion}"/>
-		          </form>
-		        </j:when>
-		      </j:choose>
-          <j:choose>
+			<j:if test="${!p.isManuallyApproved()}">
```

Comment:
It's better to use spaces separators if you change codelines. The original tabs separator seems to be incorrect